<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DJctNodePanel</class>
 <widget class="QWidget" name="DJctNodePanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>288</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DJctNodePanel</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Junction Node Properties</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout" columnstretch="1,1" rowminimumheight="20,20,20,20,20">
        <item row="2" column="1">
         <widget class="QPlainTextEdit" name="Details_plainTextEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>60</height>
           </size>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustToContents</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="Details_Label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Details</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="Box_Junction_Label">
          <property name="text">
           <string>Box Junction</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="Signalized_comboBox">
          <property name="editable">
           <bool>false</bool>
          </property>
          <property name="currentText">
           <string>False</string>
          </property>
          <property name="insertPolicy">
           <enum>QComboBox::NoInsert</enum>
          </property>
          <item>
           <property name="text">
            <string>False</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>True</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="NameText_Label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="Signalized_Label">
          <property name="text">
           <string>Signalized</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="IDText_Label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QComboBox" name="Box_Junction_comboBox">
          <property name="currentText">
           <string>False</string>
          </property>
          <property name="insertPolicy">
           <enum>QComboBox::NoInsert</enum>
          </property>
          <item>
           <property name="text">
            <string>False</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>True</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="ID_Label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="Name_LineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>Name_LineEdit</tabstop>
  <tabstop>Details_plainTextEdit</tabstop>
  <tabstop>Signalized_comboBox</tabstop>
  <tabstop>Box_Junction_comboBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
